### DISCUSSION TITLE
v0.9.0 🎉

### DISCUSSION THREAD

**Discussion by 4gray** [4gray/iptvnator#89] [General]
# [0.9.0](https://github.com/4gray/iptvnator/compare/v0.8.0...v0.9.0) (2021-10-14)

![reorder playlists](https://raw.githubusercontent.com/4gray/iptvnator/master/src/assets/updates/090/reorder-playlists.gif)

### Bug Fixes

* search feature should not affect favorites list ([ef52f77](https://github.com/4gray/iptvnator/commit/ef52f77c117c644c2173d4b82783028a19f25011)), closes [#71](https://github.com/4gray/iptvnator/issues/71)

### Features

* add chinese translation 🇨🇳 ([a497f05](https://github.com/4gray/iptvnator/commit/a497f0570175618d7053b53fd47aa907e6361f17))
* global subtitle display setting ([4d2e175](https://github.com/4gray/iptvnator/commit/4d2e17565d247c2a6bc9ae3d23ab37ff52033478))
* rearrange the display order of playlists ([757c739](https://github.com/4gray/iptvnator/commit/757c739d92d0a646f1927a4c3f2d3eb8425876df)), closes [#77](https://github.com/4gray/iptvnator/issues/77)

<hr /><em>This discussion was created from the release <a href='https://github.com/4gray/iptvnator/releases/tag/v0.9.0'>v0.9.0 🎉 </a>.</em>

**Comment by what-a** [comment]
Love the new features and fixes!   Thank you! 👏👏👏

**Comment by martjay** [comment]
Can you add the function of automatic translation of subtitles? Because some people can't read English. LIke youtube

**Comment by fraesco123** [comment]
missing-dependency-on-libc needed by opt/iptvnator/chrome-sandbox and 8 others
E: iptvnator: no-copyright-file

**Comment by panilssen** [comment]
Trying this app on Debian 11 using appimage.
When adding my m3u URL (including username & pw) nothing happens. It will not add the URL.
Is this because of username and pw?
Same URL works fine on GSE IPTV app on my OS X laptop.

**Reply by what-a** [reply]
See issue #53 
Xtreme API playlists are not currently supported in IPTVnator.

**Comment by dragonking91** [comment]
Can you add HW and HW+ decode, please?

**Reply by fraesco123** [reply]
Always enable hardware acceleration On BIOS, This will usually be a higher frame rate (the number of images displayed per second), and the higher the frame rate the smoother the animation. Better performance of your application.
Devices may not support HW+ yet and some videos may not be played properly and any Hardware Software. Have commercial right

**Reply by fraesco123** [reply]
I have no idea in open sources software.

**Reply by fraesco123** [reply]
In my case, I have testing this Intel Support from SoC Blockset,  this is big explanation to resolve Intel Support from SoC Blockset
